fpdc/settings/base.py

```diff
@@ -94,12 +95,18 @@
 # https://docs.djangoproject.com/en/1.11/ref/settings/#auth-password-validators
 
 AUTH_PASSWORD_VALIDATORS = [
-    {"NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator"},
+    {
+        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator"
```

Comment:
Oh that would be helpful. Maybe a sample `.vscode/settings.json`?

Reply:
yes that could do it, also this will be enforced by the CI [see here](https://github.com/fedora-infra/fpdc/pull/17/files#diff-b91f3d5bd63fcd17221b267e851608e8R21) once we have a CI :laughing:

Reply:
Looks like adding

```
    "python.linting.flake8Args": [
        "--max-line-length=100"
    ]
```

to the `flake8` linting calls might be helpful too, so that `black` and `flake8` are in agreement about line length.